### ISSUE TITLE
bi_connector settings don't work in mongodbatlas_advanced_cluster

### ISSUE THREAD

**Issue by ibelikov** [mongodb/terraform-provider-mongodbatlas#893]
### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
Terraform v1.3.3
+ provider registry.terraform.io/mongodb/mongodbatlas v1.4.6
```

### Terraform Configuration File

```hcl
resource "mongodbatlas_advanced_cluster" "test" {
  project_id                  = var.project_id
  name                        = var.cluster_name
  cluster_type                = "REPLICASET"
  version_release_system      = "CONTINUOUS"
  encryption_at_rest_provider = "AWS"
  replication_specs {
    ...
  }
  bi_connector {
    enabled         = true
    read_preference = "secondary"
  }
}
```

### Actual Behavior

Terraform apply will finish successfully but BI Connector will not be enabled (can be verified in the Atlas cloud portal).
Each subsequent run of terraform plan/terraform apply will show pending changes to enable bi connector:

```hcl
Terraform will perform the following actions:

  # module.mongodb_atlas.mongodbatlas_advanced_cluster.backbone will be updated in-place
  ~ resource "mongodbatlas_advanced_cluster" "test" {
        id                          = ...
        name                        = ...
        # (15 unchanged attributes hidden)

      ~ bi_connector {
          ~ enabled         = false -> true
            # (1 unchanged attribute hidden)
        }

        # (2 unchanged blocks hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

### Additional Context

mongodbatlas_advanced_cluster calls expandBiConnectorConfig function (https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/mongodbatlas/resource_mongodbatlas_advanced_cluster.go#L305) which tries to read bi_connector_config resource block (https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/mongodbatlas/resource_mongodbatlas_cluster.go#L499), while the block name in mongodbatlas_advanced_cluster resource is still bi_connector (no _config).

**Comment by martinstibbe** [comment]
@ibelikov Verified it seems to ignore setting on apply (if I set it via portal and it reads it back as set in statefile.) I will continue to investigate to determine cause of issue.

**Comment by Zuhairahmed** [comment]
@ibelikov we confirmed this is a bug and added to future sprint to resolve. Should have fix out in next month or two and will keep you updated on this issue. Thank you. Internal ticket [INTMDB-455](https://jira.mongodb.org/browse/INTMDB-455)

**Comment by Zuhairahmed** [comment]
@ibelikov as an update, this issue has been picked up in current sprint cycle and you can expect fix to be included as part of v1.8.0 release next week. will let you know when latest release has been published to the Terraform Registry.

**Comment by Zuhairahmed** [comment]
@ibelikov we just released v1.8.0, feel free to give it try! issue should have been resolved.

**Comment by Zuhairahmed** [comment]
closing this issue, but feel free to re-open if you need anything else
